lib/ex_doc/formatter/html/templates.ex

```diff
@@ -84,12 +84,30 @@ defmodule ExDoc.Formatter.HTML.Templates do
     for extra <- extras do
       %{id: id, title: title, group: group, content: content} = extra
 
-      %{
-        id: to_string(id),
-        title: to_string(title),
-        group: to_string(group),
-        headers: extract_headers(content)
-      }
+      item =
+        %{
+          id: to_string(id),
+          title: to_string(title),
+          group: to_string(group),
+          headers: extract_headers(content)
+        }
+
+      case extra do
+        %{search_data: search_data} when is_list(search_data) ->
+          search_data =
+            Enum.map(search_data, fn search_item ->
+              %{
+                anchor: search_item.anchor,
```

Comment:
With the removal of `:description`, `Map.take` and `Map.put` here didn't make sense anymore.